.vscode/settings.json

```diff
@@ -17,5 +17,8 @@
 		"**/.DS_Store": true,
 		"**/node_modules": false,
         ".cache/": true,
+	},
+	"[javascript]": {
+		"editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
@Aibono1225 this changes the default formatted for all developers.

Are you ok with this change?

Reply:
Sorry I commited the wrong code, this commit has been revert now😊